src/services/BlockchainService/schemas/infoSchema.ts

```diff
@@ -11,7 +11,7 @@ const infoSchema = z.object({
   chainId: z.number(),
   chainName: z.string(),
   exchangeContractAddress: z.string(),
-  swapExecutorContractAddress: z.string().optional(),
+  swapExecutorContractAddress: z.string(),
```

Comment:
Are all our environments have swapExecutorContractAddress? If we declare this as required we should be sure values are present in all envs.